.github/workflows/ci.yml

```diff
@@ -129,19 +129,19 @@ jobs:
       # Run the installer script
     - name: Install dependencies
       run: |
-        node . install --ignore-scripts --no-audit
-        node . rebuild
+        node ./bin/npm-cli.js install --ignore-scripts --no-audit
+        node ./bin/npm-cli.js rebuild
 
     - name: Run workspaces tests
-      run: node . test -w ./packages -- --no-check-coverage -t600 -Rbase -c
+      run: node ./bin/npm-cli.js test -w ./packages -- --no-check-coverage -t600 -Rbase -c
       env:
         DEPLOY_VERSION: testing
 
   build:
     strategy:
       fail-fast: false
       matrix:
-        node-version: ['10.1', 10.x, '12.1', 12.x, '14.1', 14.x, '16.1', 16.x]
+        node-version: ['12.13.0', 12.x, '14.15.0', 14.x, '16.1', 16.x]
```

Comment:
yes - because there was still breakage in the .0 builds, due to a super weird bug with (iirc) Object.entries and tap's matching assertions - and even though npm works perfectly there, it wasn't worth fixing tap to avoid that bug.

Reply:
Did that include 16.0?

Reply:
no, it was only 12.0 - i have no idea why 14.0 and 16.0 were skipped, but i assume it was just cargo-culted.